src/app/globals.css

```diff
@@ -25,3 +25,42 @@ body {
     )
     rgb(var(--background-start-rgb));
 }
+
+.h1-landing-text {
```

Comment:
[Please Change]: https://github.com/ufssd/ufssd-website/pull/11#discussion_r1361417772 still applies - these styles should be in a separate `Landing.module.css` file, like with [`About.module.css`](https://github.com/ufssd/ufssd-website/pull/12/files#diff-9ce4959ddd738600ac6c0f6fa23fb0c4682422372fd29325b30d54d216a5b349).

[Please Change]: As part of these, these should use `camelCase` names as per https://github.com/ufssd/ufssd-website/pull/12#discussion_r1364213963.

Reply:
@WillBAnders, you probably want it named `Home.module.css`. I remember in an earlier conversation that CSS modules would either be named `page.module.css` or named after the component that it styles. In this case, the component is named `Home`, not `Landing`.

Reply:
Moved to Landing.module.css and changed to camelCase names